### DISCUSSION TITLE
More than one livestream possible on one card

### DISCUSSION THREAD

**Discussion by oxygi** [MiczFlor/RPi-Jukebox-RFID#2275] [Q&A]
I followed the instructions and got a stream running in livestream.txt. :-)
Question 1: Can I put more than one stream on one card?
Question 2: How can I play an audio file before each new stream? Since I don't have a display, I would like to have the radio station announced - resp. I would record the name of the radio and run the file before playing the stream so that I know which stream is coming up.

I am still on version [v2.5.0]
Thank you very much in advance!

**Comment by s-martin** [comment]
> I followed the instructions and got a stream running in livestream.txt. :-)
> Question 1: Can I put more than one stream on one card?

Yes, I think so (never did it myself though).
https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/MANUAL#adding-webradio-station-and-other-online-streams

Be aware that a real **livestream** never ends, so a file AFTER a livestream will never be played.

> Question 2: How can I play an audio file before each new stream? Since I don't have a display, I would like to have the radio station announced - resp. I would record the name of the radio and run the file before playing the stream so that I know which stream is coming up.

Yes, you can mix local files and streams, see https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/MANUAL#mixing-audio-files-and-web-streams

Again file after real livestream will never be played.
